scripts/common/utilbinaries.sh

```diff
@@ -326,7 +326,7 @@ function apply_iptables_config() {
 
 function is_ha() {
     local master_count=
-    master_count="$(kubernetes_masters | wc -l)"
+    master_count="$(maybe kubernetes_masters | wc -l)"
```

Comment:
what the heck is 'maybe'?

Reply:
https://github.com/replicatedhq/kURL/blob/9a6d89fea7940e29a83c058f1684bd6be1ad158c/scripts/common/common.sh#L892-L897

looks good